docs/how_to_build_images.md

```diff
@@ -0,0 +1,36 @@
+# How to build images
+
+Use appropriate playbook inside `playbooks` directory to build the images you need.
+
+To build localhost/foreman:2.1-42 image:
+
+```
+ansible-playbook playbooks/build-foreman-image.yml -e version=2.1 -e version_build=42
```

Comment:
I wonder if it would help to name it `release` or `build_release` instead of `version_build` to mirror how we think about RPMs? If not, I'd consider reversing the terms to be `build_version` so it reads more naturally.

Reply:
Renamed.